test/mocks/server/bootstrap_extension_factory.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <chrono>
+#include <cstdint>
+#include <list>
+#include <string>
+
+#include "envoy/server/bootstrap_extension_config.h"
+
+#include "gmock/gmock.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+class MockBootstrapExtensionFactory : public BootstrapExtensionFactory {
+public:
+  MockBootstrapExtensionFactory();
+  ~MockBootstrapExtensionFactory() override;
+
+  MOCK_METHOD(BootstrapExtensionPtr, createBootstrapExtension,
+              (const Protobuf::Message&, Configuration::ServerFactoryContext&), (override));
+  MOCK_METHOD(ProtobufTypes::MessagePtr, createEmptyConfigProto, (), (override));
+  MOCK_METHOD(std::string, name, (), (const, override));
+};
+} // namespace Configuration
+
```

Comment:
Removing the blank lines between closing namespace brackets would be more consistent with the opening namespace brackets.

Reply:
Now they're removed